lib/features/auth/cubit/auth_cubit.dart

```diff
@@ -54,6 +54,17 @@ class AuthCubit extends Cubit<AuthState> {
       );
 
       var userExt = await _authRepositoy.fetchUserExtension(session.userGUID);
+      if (password == TempUser.defaultPassword) {
+        await AnalyticsHelper.setUserProperties(userId: userExt.guid);
+        unawaited(
+          AnalyticsHelper.logEvent(
+            eventName: AmplitudeEvents.continueByEmailSignUpTempUserClicked,
+            eventProperties: {
+              'country': userExt.countryCode,
```

Comment:
Can you make this profile_country instead of country? Amplitude will overwrite the country attribute.

Reply:
Oki 👌